src/graphdb.js

```diff
@@ -241,4 +335,4 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
 }
 
 
-export { graphDBInferenceSchema };
\ No newline at end of file
+export { graphDBInferenceSchema, replaceCleanseLabel };
```

Comment:
In general it is not a good idea to export a function just for usage in unit tests. The better way would be to have the unit test verify the functionality of the exported function - the `graphDBInferenceSchema` function.